app/src/main/java/com/trunghoang/aroundhere/ui/discover/DiscoverPresenter.java

```diff
@@ -0,0 +1,49 @@
+package com.trunghoang.aroundhere.ui.discover;
+
+import android.support.annotation.NonNull;
+
+import com.trunghoang.aroundhere.data.model.Place;
+import com.trunghoang.aroundhere.data.model.PlaceDataSource;
+import com.trunghoang.aroundhere.data.model.PlaceRepository;
+
+import java.util.List;
+
+public class DiscoverPresenter implements DiscoverContract.Presenter {
+    private final PlaceRepository mPlacesRepository;
+    private final DiscoverContract.View mPlacesView;
```

Comment:
```suggestion
    private final DiscoverContract.View mView;
```
nếu chỉ có 1 view thì e có thể đặt tên ntn cho dễ đọc